SchedulerJobs/SchedulerJobs.Services/CloseConferenceService.cs

```diff
@@ -31,6 +34,29 @@ public async Task<int> CloseConferencesAsync()
 
             return conferenceCount;
         }
+
+        public async Task<int> DeleteAudiorecordingApplicationsAsync()
+        {
+            var conferences = await _videoApiService.GetExpiredAudiorecordingConferences();
+            var conferencesCount = 0;
+            if (conferences != null && conferences.Any())
+            {
+                conferencesCount = conferences.Count;
+                conferences.ForEach(async s =>
+                {
+                    try
+                    {
+                        await _videoApiService.DeleteAudiorecordingApplication(s.HearingId);
+                    }
+                    catch (Exception ex)
+                    {
+                        conferencesCount--;
```

Comment:
Not thread safe

Reply:
object not being thread-safe is when a single instance (or static) is accessed by multiple threads at the same time. Do we?